src/iqm/iqm_client/iqm_client.py

```diff
@@ -854,8 +858,93 @@ def get_quantum_architecture(self, *, timeout_secs: float = REQUESTS_TIMEOUT) ->
         self._architecture = qa
         return qa
 
+    def get_quality_metric_set(
+        self, calibration_set_id: UUID | None = None, *, timeout_secs: float = REQUESTS_TIMEOUT
+    ) -> QualityMetricSet:
+        """Retrieve the latest quality metric set from the server using the V1 API
+        (Cocos circuits execution and Resonance) architecture.
```

Comment:
```suggestion
        """Retrieve the latest quality metric set from the server for the given calibration set.
```
The `self._api.url()` construct can provide handle any API version (that supports the QUALITY_METRICS endpoint), right?

Reply:
Yes, there are tests for the different API versions.